runhouse/resources/folders/s3_folder.py

```diff
@@ -61,18 +57,20 @@ def _download(self, dest):
         # https://github.com/skypilot-org/skypilot/blob/983f5fa3197fe7c4b5a28be240f7b027f7192b15/sky/data/storage.py#L231
         remote_dir = self.path.lstrip("/")
         remote_dir = f"s3://{remote_dir}"
-        subprocess.run(
-            ["aws", "s3", "sync", remote_dir, dest],
-            stdout=subprocess.DEVNULL,
-            stderr=subprocess.DEVNULL,
-            check=True,
-        )
-
-    def _download_command(self, src, dest):
-        from sky.cloud_stores import S3CloudStorage
+        try:
+            subprocess.run(
+                ["aws", "s3", "sync", remote_dir, dest],
+                stdout=subprocess.DEVNULL,
+                stderr=subprocess.DEVNULL,
+                check=True,
+            )
+        except subprocess.CalledProcessError as e:
+            raise e
 
-        download_command = S3CloudStorage().make_sync_dir_command(src, dest)
-        return download_command
+    def _download_command(self, src: str, dest: str):
+        # https://github.com/skypilot-org/skypilot/blob/983f5fa3197fe7c4b5a28be240f7b027f7192b15/sky/cloud_stores.py#L133
```

Comment:
should this be https://github.com/skypilot-org/skypilot/blob/983f5fa3197fe7c4b5a28be240f7b027f7192b15/sky/cloud_stores.py#L68?